mesonbuild/mintro.py

```diff
@@ -29,6 +29,8 @@
 def add_arguments(parser):
     parser.add_argument('--targets', action='store_true', dest='list_targets', default=False,
                         help='List top level targets.')
+    parser.add_argument('--target-batch', action='store_true', dest='target_batch', default=False,
+                        help='Batch export info on all targets.')
```

Comment:
Would '--targets-all' or similar be a better name for this, since that's what it does?

Reply:
It is meant to export everything about a target needed by an IDE, not just files. We'd need to get some feedback from IDE devs. (For some reason github prevents me from tagging TH of Qt Creator on this post.)